src/tests/components/AdicionarUsuarioDialog.spec.tsx

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable testing-library/no-unnecessary-act */
+/* eslint-disable testing-library/render-result-naming-convention */
+import { render, waitFor, act, fireEvent } from "@testing-library/react";
+// import { Permissao } from "../models/auth";
```

Comment:
Remover cometários desnecessários
```suggestion
// import { Permissao } from "../models/auth";
```

Reply:
Comentários removidos